internal/pkg/otel/components.go

```diff
@@ -113,7 +112,6 @@ func components(extensionFactories ...extension.Factory) func() (otelcol.Factori
 			elasticinframetricsprocessor.NewFactory(),
 			resourcedetectionprocessor.NewFactory(),
 			memorylimiterprocessor.NewFactory(),
-			lsmintervalprocessor.NewFactory(),
```

Comment:
Does this mean that the `lsminterval` won't be available from the configuration?

Reply:
Do we consider breaking changes for tech preview?

Reply:
FWIW, the fix has already been released for [8.18](https://github.com/elastic/elastic-agent/pull/7452) and [9.x](https://github.com/elastic/elastic-agent/pull/7172), this PR is only to update main branch - not exactly sure what release would that correspond to... maybe 8.19?

Reply:
Ahh, good catch. I was not aware of those changes. Resolving thread.